docs/samples/python/customizing_presidio_analyzer.ipynb

```diff
@@ -0,0 +1,677 @@
+{
```

Comment:
![Screen Shot 2021-03-11 at 13 40 57](https://user-images.githubusercontent.com/1862328/110782325-abc92900-826f-11eb-8531-2be5088f9b0f.png)
First lines are about the plugin... should it be this way?

Reply:
No... good catch! fixing